libkbfs/block_retrieval_queue.go

```diff
@@ -66,6 +69,14 @@ type blockPtrLookup struct {
 	t  reflect.Type
 }
 
+// blockRetriever specifies a method for retrieving blocks asynchronously
+type blockRetriever interface {
```

Comment:
Understood. Will move to the prefetcher (it seems that `interfaces.go` is mainly for exported interfaces).